From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ziqing Hui <zhui@codeweavers.com>
Date: Wed, 22 Mar 2023 15:41:00 +0800
Subject: [PATCH] kernelbase: HACK: Force desktop GL for PaladinLias.

CW-Bug-Id: #20816
Valve-commit: 9ab8f9b271f81eff5d8966d69a2249501f31789c
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index bb97d18..635fa2a 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -517,6 +517,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"nw.exe.exe", L" --use-angle=d3d9"},
         {L"DC Universe Online\\LaunchPad.exe", L" --use-gl=swiftshader"},
         {L"PlanetSide 2\\LaunchPad.exe", L" --use-gl=swiftshader"},
+        {L"PaladinLias\\Game.exe", L" --use-gl=desktop"},
     };
     unsigned int i;
 
